hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -152,6 +152,10 @@
       name: en_ibex_tracer
       build_opts: ["+define+RVFI=1"]
     }
+    {
+      name: pad_ctrl_test_mode
```

Comment:
nit: Can you add a comment explaining what this mode is for? Maybe something like...

```suggestion
    // Build mode that configures a testbench in a manner suitable for pad
    // testing. For example, the testbench may need to have different agents
    // or interfaces connected to the pads for marked tests.
    {
      name: pad_ctrl_test_mode
```

Reply:
thx.  Added.